src/domain/entities/AttributeValue.ts

```diff
@@ -1,3 +1,3 @@
-import { Ref } from "@eyeseetea/d2-api";
+import { Ref } from "./Ref";
 
 export type AttributeValue = { attribute: Ref; value: string };
```

Comment:
An attribute is a DHIS2 implementation detail, removed the domain. I see that's only used by D2DataElement, so it makes sense.